doc/en/user/source/extensions/wps-download/mapAnimationDownload.rst

```diff
@@ -215,7 +217,87 @@ The ``formattedTimestamper`` decoration ensures the frame time is included in th
       </decoration>
     </layout>
 
+Sample variable-based DownloadAnimation request
++++++++++++++++++++++++++++++++++++++++++++++++
 
+If you want to animate a map based on variables, proceed as follows:
+
+* specify the ``animateParam`` variable. You can use this to animate SLD variables, as described in ``Variable substitution in SLD``
+  or animate through a custom parameter
+* specify the ``animateValues`` parameter as a comma separated list of your values that will be used in the animation
```

Comment:
We have parameters whose value is comma-separated, or can contain a comma. The documentation should specify a way to escape commas (e.g., see the [old animator docs for reference](https://docs.geoserver.org/stable/en/user/tutorials/animreflector.html#overview)).